engine/combat/combat_controller.py

```diff
@@ -130,14 +132,19 @@ def _encounter_controller_factory(self: Self) -> EncounterController:
             case CombatEncounter.LiveManaTutorial:
                 return LiveManaTutorialController()
             case _:
-                match map(lambda x: x.guid, combat_manager.enemies):
+                match list(map(lambda x: x.guid, combat_manager.enemies)):
```

Comment:
for some reason the map was consumed and couldn't be used in the second case ... not sure what python sorcery caused this, but wrapping this in list fixes it.